src/components/AddContent/components/ContentValidity.tsx

```diff
@@ -0,0 +1,71 @@
+import { Flex, FlexItem, Spinner } from '@patternfly/react-core';
+import { FormikErrors, FormikTouched } from 'formik';
+import { global_success_color_100, global_danger_color_100 } from '@patternfly/react-tokens';
+import { CheckCircleIcon, ExclamationCircleIcon } from '@patternfly/react-icons';
+import { createUseStyles } from 'react-jss';
+const red = global_danger_color_100.value;
+const green = global_success_color_100.value;
+
+const useStyles = createUseStyles({
+  red: { color: red, fontWeight: 'bold', fontSize: '14px' },
+  green: { color: green, fontWeight: 'bold', fontSize: '14px' },
+});
+
+interface Props {
+  touched?: FormikTouched<{
+    name: string;
+    url: string;
+    gpgKey: string;
+    gpgLoading: boolean;
+    expanded: boolean;
+  }>;
+  errors?: FormikErrors<{
+    name: string;
+    url: string;
+    gpgKey: string;
+    gpgLoading: boolean;
+    expanded: boolean;
+  }>;
+  loading?: boolean;
+}
+
+const ContentValidity = ({ touched, errors, loading }: Props) => {
```

Comment:
If I'm creating with a URL where I get "no such host", should it block me from creating or just give me a warning? I'm not sure what causes that error, but what if the URL is just down momentarily and I know it's valid?

Reply:
This section of the code it not necessarily relevant, just clicked here to get the conversation UI

Reply:
That's what I meant by this piece, I think the general consensus is that we want to stop the users (as per design feedback thus far).
![Screen Shot 2022-08-15 at 2 18 23 PM](https://user-images.githubusercontent.com/38083295/184711220-41043043-ad3a-46ca-bcc9-d063a20ae575.png)

Reply:
> was it intended to keep the "too long" and "too short" validation or was that just a proof of concept?

I think the too long/too short make sense, I just need feedback on limits there. Logical limits of between 2 - 50 are currently set.

Reply:
You can leave everything as-is. I was just pointing some things out in case they were being overlooked, but it sounds like they're not.